.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: Build site
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    name: Build site
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: "package.json"
+      - run: |
+          cp .env.example .env
```

Comment:
Sorry I didn't see this the first time - are we just using those hardcoded values as the nextauth_secret? I think it uses that for JWT, leaving us open to someone signing their own JWT tokens with the hardcoded signature.

~~Suggest adding it as an env variable set by a command in a Dockerfile~~

Apparently this is impossible, https://github.com/moby/moby/issues/29110, so instead suggest doing what someone that replied to that issue says to do or have next.config.js run some code to grab it from a file created with a RUN command.

This doesn't need to be in this commit, but should be addressed probably by devops as well soon.

Reply:
That's only to test whether it builds with dummy value.

In staging and production, the token will be set on runtime and will not leave the machine.